src/storage/BlockStorage.ts

```diff
@@ -244,6 +244,116 @@ export class BlockStorage {
         return true;
     }
 
+    /*────────────────────────────────────────────────────────────────────────────
+      GET ALL BLOCKS BY FORK ID
+    ────────────────────────────────────────────────────────────────────────────*/
+    getBlocksByForkId(
+        forkId: ForkId,
+        sortOrder?: "asc" | "desc"
+    ): BlockEntry[] {
```

Comment:
some iteration logic is needed, but not in the form of fetch everything and sort -> in most cases we're interested only in the `last N` blocks